tests/phpunit/tests/Admin/Cross_Origin_Isolation.php

```diff
@@ -59,6 +59,12 @@ public function setUp() {
 		$user_preferences->register();
 	}
 
+	public function tearDown() {
+		unset( $GLOBALS['current_screen'] );
+
+		parent::tearDown();
+	}
```

Comment:
As this is a general oversight of WP Core, it might make sense to put it directly in your base `TestCase` (it already is in `DependencyInjectedTestCase`).

Also, see https://core.trac.wordpress.org/ticket/53431

Reply:
Aha! _That's why_ I thought that core was already doing this.

I was checking the `WP_UnitTestCase` code locally (from trunk) and saw the code from https://core.trac.wordpress.org/changeset/51419 there, wondering why I still had to do it manually.